### PR TITLE
introduce "trusted" tier for adapters

### PR DESCRIPTION
**Pull request by dataders**
## What are you changing in this pull request and why?

Introducing Trusted adapters.

### Supported Data Platforms

[preview](https://deploy-preview-3758--docs-getdbt-com.netlify.app/docs/supported-data-platforms)

- Added bullet point about "Trusted adapters"
- Added Trusted adapter cards below the verified adapter cards

### Trusted Adapters

[preview](https://deploy-preview-3758--docs-getdbt-com.netlify.app/docs/trusted-adapters)

- brand new page that gives overview
- links out to a new adapter dev guide

### Building a Trusted Adapter

[preview](https://deploy-preview-3758--docs-getdbt-com.netlify.app/guides/dbt-ecosystem/adapter-development/8-building-a-trusted-adapter)

- details what Trusted adapter means
- enumerates the relevant guidelines
- explains how to become Trusted

### New Issue Template

[Link to `.yaml`](https://github.com/dbt-labs/docs.getdbt.com/pull/3758/files#diff-ef4a0e442faae77613880dfc0527a87b5f0444e3512597c3d622eaed7ce5ffaf) (there's no way to preview this AFAICT)

This serves as both a webform and also a "contract"

### Other changes

- Make the cards view for Verified and Trusted adapters into snippets for re-usability
- add Verified Adapters overview page to sidebar
- remove outdated reference on "Connect Adapters" page
- small cleanups

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] internal stakeholders approve

Adding new pages (delete if not applicable):
- [x] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

### CODE REVIEW

**Review by amyd123 — APPROVED** [review]
This looks great - excited to see this go live.

File: .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      placeholder: your preferred email and/or dbt Slack handle
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: author_type
+    attributes:
+      label: Which of these best describes you?
+      options:
+        - label: I am a dbt Community member or partner contributing to the Developer Blog
```
Comment (jasnonaz) [inline .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml]: update - not sure if we should delete this or tweak it

File: website/snippets/_adapters-trusted.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div className="grid--4-col">
+
+<Card
+    title="Athena*"
```
Comment (jasnonaz) [inline website/snippets/_adapters-trusted.md]: We should validate this list before go-live

File: .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+name: Add adapter to Trusted list
+description: >
+  For adapter maintainers who wish to have theirs added to the list of [Trusted adapters](https://docs.getdbt.com/docs/trusted-adapters)
+labels: ["adapter maintainers","developer blog"]
```
Comment (jasnonaz) [inline .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml]:
```SUGGESTION 
labels: ["adapter maintainers"]

File: .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We're excited that you'd like to support your adapter formally as "Trusted"! This template will ensure that you are aware of the process and the guidelines. Additionally, that you can vouch that your adapter currently what is expected of a Trusted adapter
```
Comment (jasnonaz) [inline .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml]:
```SUGGESTION
        We're excited that you'd like to support your adapter formally as "Trusted"! This template will ensure that you are aware of the process and the guidelines. Additionally, that you can vouch that your adapter currently meets the standards of a Trusted adapter

File: .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+  - type: checkboxes
+    id: read-program-guide
+    attributes:
+      label: Please agree to the each of the following
+      options:
+        - label: I am a maintainer of the adapter being submited for Trusted status
+        - label: I have read both the [Trusted adapters](https://docs.getdbt.com/docs/trusted-adapters) and [Building a Trusted Adapter](https://docs.getdbt.com/guides/dbt-ecosystem/adapter-development/8-building-a-trusted-adapter) pages.
+        - label: I believe that the adapter currently meets the expectations given above
+        - label: I will ensure this adapter stays in compliance with the guidelines indefinitely
```
Comment (jasnonaz) [inline .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml]:
```SUGGESTION 
- label: I will ensure this adapter stays in compliance with the guidelines

File: website/docs/docs/supported-data-platforms.md [hunk 6]
```diff
@@ -6,73 +6,32 @@ description: "Connect dbt to any data platform in dbt Cloud or dbt Core, using a
...
 
-<Card
-    title="AlloyDB"
-    body="<a href='/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb'><img src='/img/icons/dbt-bit.svg' width='7%'/>Set up in dbt Cloud </a> <br /><a href='/docs/core/connect-data-platform/alloydb-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI </a> <br /><br /><a href=https://badge.fury.io/py/dbt-postgres><img src=https://badge.fury.io/py/dbt-postgres.svg/></a>"
-    icon="alloydb"/>
-<Card
-    title="BigQuery"
-    body="<a href='/docs/cloud/connect-data-platform/connect-bigquery'><img src='/img/icons/dbt-bit.svg' width='7%'/>Set up in dbt Cloud </a> <br /><a href='/docs/core/connect-data-platform/bigquery-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI </a> <br /><br /> <a href=https://badge.fury.io/py/dbt-bigquery><img src=https://badge.fury.io/py/dbt-bigquery.svg/></a>"
-    icon="bigquery"/>
+There are three types of adapters available today. The purpose of differentiation is to provide users with an easier means to evaluate adapter quality.
```
Comment (jasnonaz) [inline website/docs/docs/supported-data-platforms.md]:
```SUGGESTION 
There are three types of adapters available today:

File: website/docs/docs/supported-data-platforms.md [hunk 7]
```diff
@@ -6,73 +6,32 @@ description: "Connect dbt to any data platform in dbt Cloud or dbt Core, using a
...
 
-<Card
-    title="Databricks"
-    body="<a href='/docs/cloud/connect-data-platform/connect-databricks'><img src='/img/icons/dbt-bit.svg' width='7%'/>Set up in dbt Cloud </a> <br /><a href='/docs/core/connect-data-platform/databricks-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI  </a> <br /><br /> <a href=https://badge.fury.io/py/dbt-databricks><img src=https://badge.fury.io/py/dbt-databricks.svg/></a>"
-    icon="databricks"/>
+- **Verified** &mdash; [Verified adapters](verified-adapters) are those that have completed a rigorous verification process in collaboration with dbt Labs.
+- **Trusted** &mdash; [Trusted adapters](trusted-adapters) are those where the adapter maintainers have agreed to meet a higher standard of quality.
```
Comment (jasnonaz) [inline website/docs/docs/supported-data-platforms.md:20]: Should we say something about how these are also open source?

File: website/docs/guides/dbt-ecosystem/adapter-development/8-trusting-a-new-adapter.md
Comment (jasnonaz) [inline website/docs/guides/dbt-ecosystem/adapter-development/8-trusting-a-new-adapter.md:1]: filename on this is still wonky

File: website/docs/guides/dbt-ecosystem/adapter-development/8-trusting-a-new-adapter.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
+---
+title: "Building a Trusted Adapter"
+id: "8-building-a-trusted-adapter"
+---
+
+The Trusted adapter program exists to allow adapter maintainers to demonstrate to the dbt community that your adapter is trusted to be used in production.
+
+## What does it mean to be trusted
+
+Below are some categories with stuff.
```
Comment (jasnonaz) [inline website/docs/guides/dbt-ecosystem/adapter-development/8-trusting-a-new-adapter.md]: I think this was just a placeholder line we can delete?

File: website/docs/guides/dbt-ecosystem/adapter-development/8-trusting-a-new-adapter.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+By opt-ing into the below, you agree to this, and we take you at your word. dbt Labs reserves the right to remove an adapter from the trusted adapter list at any time, should any of the below guidelines not be met.
```
Comment (jasnonaz) [inline website/docs/guides/dbt-ecosystem/adapter-development/8-trusting-a-new-adapter.md]:
```SUGGESTION
By opting into the below, you agree to this, and we take you at your word. dbt Labs reserves the right to remove an adapter from the trusted adapter list at any time, should any of the below guidelines not be met.

**Review by jasnonaz — CHANGES_REQUESTED** [review]
Excited to get this live! Some tweaks we can male outlined below

File: .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml (hunk 1 above)
Comment (dataders) [reply .github/ISSUE_TEMPLATE/add-adapter-to-trusted-list.yml]:
```suggestion
        - label: I am a dbt Community member
```

File: website/docs/docs/supported-data-platforms.md (hunk 7 above)
Comment (dataders) [reply website/docs/docs/supported-data-platforms.md:20]: imo, that's implied

**Review by jasnonaz — APPROVED** [review]
Good to go (with understanding we will do a fast follow on the placeholder trusted adapter components)!

**Pull request merged**